data.json

```
@@ -0,0 +1,55 @@
+{
+    "WorkBench" : 
+        {
+            "SP-6" : 270,
+            "7N9 SPP" : 297,
+            "7N12 BP" : 320,
+            "Gunpowder-Eagle" : 98,
+            "Gunpowder-Hawk" : 36
+        },
```

```suggestion
    "WorkBench": {
      "SP-6": 270,
      "7N9 SPP": 297,
      "7N12 BP": 320,
      "Gunpowder-Eagle": 98,
      "Gunpowder-Hawk": 36
    },
```

JSON은 탭 길이를 2칸, 그리고 `:` 쓸때 띄어쓰기 저렇게 하는게 컨벤션이긴 합니다!
사실 컨벤션이라 크게 중요하진 않는데 지켜주면 좋죠! 만약 바꾼다면 아래쪽도 다 같이 바꿔주세요.